### PR TITLE
global: compatibiliy mode for raven==5.1.0

### PR DESCRIPTION
**Pull request by mihaibivol**
* Adds compatibility mode for raven==5.1.0

Signed-off-by: Mihai Bivol <mihai.bivol@cern.ch>

### CODE REVIEW

**Comment by jirikuncar** [comment]
@mihaibivol can you fix a typo in commit message: `	global: compatibility mode ...`?

**Comment by mihaibivol** [comment]
@jirikuncar fixed the typo. Also I changed the build matrix to cover all `Raven` versions rather than creating a new test case.

File: tests/test_sentry.py [hunk 1]
```diff
@@ -131,7 +131,7 @@ def test_task():
         content_type='application/json',
         status=200,
     )
-    test_task()
+    test_task.delay()
```
Comment (lnielsen) [inline tests/test_sentry.py]: I don't think this should be changed - but i'm a bit surprised that the test pass :-)

**Comment by jirikuncar** [comment]
@mihaibivol can you please run `kwalitee check message upstream/master..` on your branch? The bullet points are badly formatter and our automatic script would fail during automatic release notes generation step. Thanks!

File: tests/test_sentry.py (hunk 1 above)
Comment (lnielsen) [inline tests/test_sentry.py]: Ups….my mistake…..it's correct :-)

**Comment by mihaibivol** [comment]
@jirikuncar done :)

Is it alright to propose another alpha release after this gets merged?

**Comment by jirikuncar** [comment]
@mihaibivol sure!

**Pull request merged**